src/fsharp/pars.fsy

```diff
@@ -5735,9 +5769,28 @@ identExpr:
 
   | opName
      { let m = lhs parseState
-       let mLparen = mkFileIndexRange m.FileIndex m.Start (mkPos m.StartLine (m.StartColumn + 1))
-       let mRparen = mkFileIndexRange m.FileIndex (mkPos m.EndLine (m.EndColumn - 1)) m.End
-       SynExpr.Paren(SynExpr.Ident($1), mLparen, Some mRparen, m) }
+       let ident, trivia = $1
```

Comment:
Minor thing: We should gradually try to remove long chunks from pars.fsy and put them in ParseHelpers.fs.  But this is by no means the worst of sins in that regard :)